tox.ini

```
@@ -14,15 +14,16 @@ commands=py.test {posargs:tests}
 passenv=
     PYTEST_ADDOPTS
 deps =
-    -r{toxinidir}/requirements-dev.txt
+extras = test
 basepython =
     py27: python2.7
     py34: python3.4
     py35: python3.5
     py36: python3.6
+    py37: python3.7
 
 [testenv:lint]
 basepython=python
 extras=lint
 commands=
-    flake8 {toxinidir}/py_ecc
+    flake8 --max-line-length=100 {toxinidir}/py_ecc
```

```suggestion
    flake8 {toxinidir}/py_ecc
```

Shouldn't be necessary ^

---

Agreed!